docs/cargo_features.md

```diff
@@ -50,6 +48,7 @@ The default feature set enables most of the expected features of a game engine,
 |custom_cursor|Enable winit custom cursor support|
 |debug|Enable collecting debug information about systems and components to help with diagnostics|
 |default_font|Include a default font, containing only ASCII characters, at the cost of a 20kB binary size increase|
+|gltf_animation|Enable animation support, and glTF animation loading|
```

Comment:
I feel like this should say just "Enable glTF animation loading" and implicitly depend on bevy_animation. 

As is the wording is kinda confusing, I don't know what "enable animation support" means without more context. Enables in bevy, just for glTF stuff, or ?